lib/linguist/languages.yml

```diff
@@ -3078,7 +3078,13 @@ Nim:
   color: "#37775b"
   extensions:
   - ".nim"
+  - ".nim.cfg"
+  - ".nimble"
   - ".nimrod"
+  - ".nims"
```

Comment:
There are only [52 files](https://github.com/search?utf8=%E2%9C%93&type=Code&ref=searchresults&q=extension%3A.nims+defined+NOT+nothack) matching the search query you provided, but it looks like there are many more `.nims` files, although many do not contain any code. Are there other keywords, besides `define`, we could use here?

Reply:
@pchaigno The use of `config.nims` is growing. `task` is one of the widely used keywords in those files: https://github.com/search?o=desc&q=extension%3A.nims+task+NOT+nothack&s=indexed&type=Code

I can really appreciate if this complex config.nims file of mine can be syntax highlighted: https://github.com/kaushalmodi/nim_config/blob/master/config.nims.

Reply:
> Could you also add the sample files (including sample files for new filenames) under samples/Nims/?

@timotheecour Can you add my above linked config.nims as a sample?

Reply:
[122 files now](https://github.com/search?utf8=%E2%9C%93&q=extension%3A.nims+task+OR+define&type=). Any other keywords?

Reply:
> Any other keywords?

[420 results][0] - *task OR define OR proc OR import OR exec OR switch*

[0]: https://github.com/search?q=extension%3A.nims+task+OR+define+OR+proc+OR+import+OR+exec+OR+switch

Reply:
@kaushalmodi I added your sample file

Reply:
`extension:nims NOT djfhdjfhdhf` gives me 520 repositories among 323 users, so I'd say we're okay with usage. Not sure why I didn't look before, but it looks like Nim is the only language using `.nims` as a file extension.